apps/nuxt3-ssr/pages/[schema]/ssr-catalogue/[resourceType]/[resource]/index.vue

```diff
@@ -0,0 +1,181 @@
+<script setup lang="ts">
+import { LocationQueryValue } from ".nuxt/vue-router";
+import { gql } from "graphql-request";
+import {
+  IColumn,
+  ISection,
+  ISchemaMetaData,
+  ITableMetaData,
+} from "interfaces/types";
+const config = useRuntimeConfig();
+const route = useRoute();
+const resourceName: string = route.params.resourceType as string;
+const schemaName = route.params.schema.toString();
+const metadata = await fetchMetadata(schemaName);
+
+const tableMetaDataFinderResult = metadata.tables.find(
+  (t: ITableMetaData) =>
+    t.id.toLocaleLowerCase() === resourceName.toLocaleLowerCase()
+);
+
+const tableMetaData = computed(() => {
```

Comment:
I would put the code of line 16 nested here

Reply:
done